Update office-add-ins-community-call.md (#3726)
OfficeDev/office-js-docs-pr: docs/overview/office-add-ins-community-call.md

The Microsoft Office Add-ins community call is a monthly call where you can learn more about new features, development practices, and additional information about creating Office Add-ins. The community call occurs the second Wednesday of each month at 8:00 AM Pacific Time. You can download the calendar invite at https://aka.ms/officeaddinscommunitycall.

## Agenda for November 9, 2022 call

> - Overview of Call and Presenter Introductions by Preethika Kiruveedula.

Reviewer: ```suggestion
- Overview of call and presenter introductions by Preethika Kiruveedula.
```

> - Learn more about Partial Documents in PowerPoint by John Teichman.

Reviewer: ```suggestion
- Learn more about partial documents in PowerPoint by John Teichman.
```

> - Introduction of Content Control Enhancements in Word Online by Yun Wang and Maja Damjanic.

Reviewer: ```suggestion
- Introduction of content control enhancements in Word on the web by Yun Wang and Maja Damjanic.
```

> - Addition of a search functionality for Office Add-ins by Adrian Wu.

Reviewer: ```suggestion
- Addition of search functionality for Office Add-ins by Adrian Wu.
```
